# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e16keyedit/e16-location.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- e16keyedit-0.2/Makefile.orig-rock-maint	2003-08-27 18:10:51.000000000 +0200
+++ e16keyedit-0.2/Makefile	2003-08-29 06:32:11.000000000 +0200
@@ -10,23 +10,23 @@
 BUILD    = \
 
 LIB_DIRS = \
-    -L/usr/local/lib -L/usr/X11R6/lib -L/usr/X11/lib -L/usr/dt/lib
+    -L$(E16)/lib -L/usr/X11R6/lib -L/usr/X11/lib -L/usr/dt/lib
 LIBS     = \
-    `gtk-config --libs` -lgdbm -lgdk_imlib
-CFLAGS   = \
-    -O -g -D_GNU_SOURCE $(DEVFLAGS)
+    `$(GTK_CONFIG) --libs` -lgdbm -lgdk_imlib
+EXTRA_CFLAGS   = \
+    -D_GNU_SOURCE
 #CFLAGS   = \
 #    -O -g -W -Wall -Wno-unused  -pedantic -pedantic-errors -ansi -D_GNU_SOURCE
 OPTFLAGS   = \
     -O6 -mpentium -mcpu=pentium -march=pentium -ffast-math -D_GNU_SOURCE
 INCLUDES = \
-    `gtk-config --cflags` -I..
+    `$(GTK_CONFIG) --cflags` -I..
 
 BIN      = e16keyedit
 
 # configurable stuff
-BINARY_DESTINATION = /usr/local/enlightenment/bin
-DATA_DESTINATION   = /usr/local/enlightenment/share
+BINARY_DESTINATION = $(E16)/bin
+DATA_DESTINATION   = $(E16)/share
 
 # commands to do things
 INSTALL  = install
@@ -92,14 +92,8 @@
 	$(RMRF) $(BIN) *.CKP *.ln *.BAK *.bak *.o core errs ,* *~ *.a .emacs_* tags TAGS make.log MakeOut $(DIST) $(DIST).tar $(DIST).tar.gz ./=build ./=install "#"*
 
 # how to install
-install:: e
-	@echo 
-	@echo 
-	@echo "You don't really want to do this. it doesn't work yet."
-	@echo 
-	@echo 
-#	for I in $(BIN); do $(INSTALL) -D -m 0775 $$I $(prefix)$(BINARY_DESTINATION)/$$I; done
-#	for I in $(DATA); do $(INSTALL) -D -m 0664 $$I $(prefix)$(DATA_DESTINATION)/$$I; done
+install:: 
+	for I in $(BIN); do $(INSTALL) -D -m 0775 $$I $(BINARY_DESTINATION)/$$I; done
 
 # how to make a dist tarball
 dist::
